msticpy/sectools/tiproviders/ti_provider_base.py

```diff
@@ -463,9 +473,11 @@ def _check_ioc_type(
 
 
 # pylint: disable=too-many-return-statements, too-many-branches
-def preprocess_observable(observable, ioc_type) -> SanitizedObservable:
+def preprocess_observable(
+    observable, ioc_type, require_url_encoding: bool = False
+) -> SanitizedObservable:
     """
-    Preprocesses and checks validity of observable against declared IoC type.
+    Preprocesses and checks validity observable against declared IoC type.
```

Comment:
why delete "of"?

Reply:
Honestly I have no idea. I have re-instated the of.

Reply:
Honestly have no idea, I've re-instated.